index.html

```diff
@@ -3040,110 +3040,136 @@ <h4>Semantic Interoperability</h4>
       <section>
         <h2>Integrity of Related Resources</h2>
         <p>
-When including a link to an external resource in a [=verifiable credential=],
-it is desirable to know whether the resource that is pointed to is the same at
-signing time as it is at verification time. This applies to cases where there is
-an external resource that is remotely retrieved as well as to cases where the
-[=issuer=] and/or [=verifier=] may have local cached copies of a resource.
-        </p>
-        <p>
-It is also desirable to know that the contents of the JSON-LD context(s) used in
-the [=verifiable credential=] are the same when used by both the
-[=issuer=] and [=verifier=].
-        </p>
-        <p>
-To validate that a resource referenced by a [=verifiable credential=] is the
-same at verification time as it is at issuing time, an implementer MAY include a
-property named <code id="defn-relatedResource">relatedResource</code> that
-stores an array of objects that describe additional integrity metadata about
-each resource referenced by the [=verifiable credential=]. If
-`relatedResource` is present, there MUST be an object in the array
-for each remote resource for each context used in the verifiable credential.
+When including a link to an external resource in a [=verifiable credential=], it
+is desirable to know whether the resource has been modified after the
+[=verifiable credential=] was issued. This applies to cases where there is an
+external resource that is remotely retrieved, as well as to cases where the
+[=issuer=] and/or [=verifier=] might have local cached copies of a resource. It
+is also desirable to know that the contents of the JSON-LD context(s) used in
+the [=verifiable credential=] are the same when used by both the [=issuer=] and
+[=verifier=].
         </p>
+
         <p class="issue" title="Mandatory listing of contexts in relatedResouce are under debate.">
 The requirement that contexts be listed in `relatedResource` is currently being
 debated in the VCWG. This requirement might be removed in future iterations of
 the specification.
         </p>
+
         <p>
-Each object in the `relatedResource` array MUST contain the
-following: the [[URL]] to the resource named `id` and the
-<code id="defn-digestSRI">digestSRI</code> information for the resource
-constructed using the method specified in
-<a href="https://www.w3.org/TR/SRI/#integrity-metadata">Subresource Integrity</a>.
+To extend integrity protection to a related resource, an [=issuer=] of a
+[=verifiable credential=] MAY include the `relatedResource` property:
         </p>
+
+        <dl>
+          <dt id="defn-relatedResource">relatedResource</dt>
+          <dd>
+The value of the `relatedResource` property MUST be associated with one or
+more objects of the following form:
```

Comment:
I think I've gotten lost in the RDF/JSON-LD terminology here. As I understand it, the `relatedResource` property appears in a [claim](https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1484.html#claims) that has a subject and a value. In this document, most uses of "X is associated with Y" appear to mean that there's a claim in which Y is the property and X is the value, although I think I see [some](https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1484.html#types) where Y is the subject instead.

If we interpret "associated with" to just mean "appears in the same claim as", then this sentence means that there's one claim (X, `relatedResource`, Y), and then there must be other claims of the form (Y, P, Z) where Z is an object with `id`, `mediaType`, etc. fields. Do you actually just mean that Y must be an object with those properties?

We might have another editorial issue that "associated with" should generally be replaced with better-defined terms.

Reply:
Hrm, "associated with" in this case is meant to mean: "The value of the `relatedResource` property MUST be one or more objects of the following form". IOW, I think we can just delete the word "associated with" here and be fine.